workflows/flows/assemble_study_tasks/make_samplesheets.py

```diff
@@ -114,6 +130,20 @@ def make_samplesheet(
             "assembly_memory": SamplesheetColumnSource(
                 lookup_string="id", renderer=lambda _: memory
             ),
+            "contaminant_reference": SamplesheetColumnSource(
+                lookup_string="id", renderer=lambda _: contaminant_reference or ""
+            ),
+            # The following 2 fields are needed in the sampleshseet, but the production setup of the pipeline
+            # sets these for the whole samplesheet. Also, if the human_reference global parameter and human_reference
+            # in the samplesheet are empty, the pipeline will fail (the user needs to provide a
+            # skip_human_decontamination flag).
+            # Reference doc -> https://github.com/EBI-Metagenomics/miassembler?tab=readme-ov-file#usage
+            "human_reference": SamplesheetColumnSource(
+                lookup_string="id", renderer=lambda _: ""
+            ),
+            "phix_reference": SamplesheetColumnSource(
+                lookup_string="id", renderer=lambda _: ""
+            ),
```

Comment:
We should probably change `SamplesheetColumnSource` to allow const rendering like

```python
class SamplesheetColumnSource(BaseModel):
    lookup_string: str | list[str] = Field(default="id")
    renderer: Callable[[Any], str] = Field(lambda x: x)
    const: Any | None = Field(None)
```

because it is a bit more readable to do

```python
            "assembly_memory": SamplesheetColumnSource(
                const=memory
            ),
            "human_reference": SamplesheetColumnSource(
                const=""
            ),
```

-> Does not need to be in this pr.

Reply:
haha I wrote one of those and the roll it back